examples/kitchen-sink.yaml

```diff
@@ -15,6 +15,11 @@ file_format: "0.1"
 # Environment variable: OTEL_SDK_DISABLED
 disabled: false
 
+# Configure the log level used by the SDK logger
```

Comment:
What is the SDK logger? I think you mean to say the internal logger used internally by the SDK.

Reply:
Copied from 
https://github.com/open-telemetry/opentelemetry-specification/blob/d6977d20eb22b3467fc5ddab43ac94bffa38ccea/specification/configuration/sdk-environment-variables.md?plain=1#L117

Reply:
I know but I think that definition is lame. Can / should we do better?

Reply:
Updated, PTAL